README.md

```diff
@@ -41,7 +41,7 @@ We may add third-party filters to AdGuard Filters Registry. When making a decisi
     * `description` — filter description
     * `timeAdded` — time when this filter was added to the registry; milliseconds since January 1, 1970; you can exec `new Date().getTime()` in the browser console to get the current time
     * `homepage` — filter website/homepage
-    * `expires` — filter's default expiration period
+    * `expires` — filter's default expiration period; used as filter update interval if "Default" is chosen for according setting in AdGuard product
```

Comment:
@PiQuark6046 Please take a look. Users may change auto-update interval.

Reply:
I know.
But, I think almost users do not know the feature as browsing Korean online community.